### ISSUE TITLE
Help, please?

### ISSUE THREAD

**Issue by Menjac123** [chaplin89/ClarifierEx#2]
https://i.imgur.com/OgZKQO4.png

**Comment by chaplin89** [comment]
I guess you should clone with --recurse because it require the source of ConfuserEx.
Anyhow please consider that this is a test. There's a lof of test code and is not guaranteed to work properly.

**Comment by Fukubus** [comment]
> I guess you should clone with --recurse because it require the source of ConfuserEx. Anyhow please consider that this is a test. There's a lof of test code and is not guaranteed to work properly.

Yes you have to clone with --recursive so that it clones ConfuserEx as well.
